collect/central_collector.go

```diff
@@ -459,7 +457,7 @@ func (c *CentralCollector) makeDecision(ctx context.Context) error {
 	if len(tracesIDs) == 0 {
 		return nil
 	}
-	statuses, err := c.Store.GetStatusForTraces(ctx, tracesIDs)
+	statuses, err := c.Store.GetStatusForTraces(ctx, tracesIDs, centralstore.AwaitingDecision)
```

Comment:
Should the check below now error? This function should no longer ever return something that isn't Awaiting